### ISSUE TITLE
[YSQL] TablegroupRemoteBootstrapTest failure

### ISSUE THREAD

**Issue by timothy-e** [yugabyte/yugabyte-db#11121]
### Description

```
ybd tsan --cxx-test integration-tests_create-table-itest --gtest_filter CreateTableITest.TablegroupRemoteBootstrapTest
```

**Comment by timothy-e** [comment]
The same issue as https://github.com/yugabyte/yugabyte-db/issues/4871

**Comment by timothy-e** [comment]
Introduced by https://phabricator.dev.yugabyte.com/D14193#04c6715f

**Comment by m-iancu** [comment]
Fixed by 7454da4.
